Library/ViewModels/SettingsPrivacyViewModel.swift

```diff
@@ -62,6 +61,8 @@ SettingsPrivacyViewModelInputs, SettingsPrivacyViewModelOutputs {
           .materialize()
     }
 
+    let updatedFetchedUser = updateEvent.values()
```

Comment:
This fixed a race condition where the notification for `.ksr_userUpdated` was firing *before* the user was actually updated on the server. Spent an infuriating hour looking for this bug.